cmd/nerdctl/main.go

```diff
@@ -291,3 +307,26 @@ func HandleExitCoder(err error) {
 		os.Exit(exitErr.ExitCode())
 	}
 }
+
+func InitConfig(cmd *cobra.Command) (err error) {
+	cfgFile, err := cmd.Flags().GetString("config-file")
+	if err != nil {
+		return err
+	}
+	if FileExist(cfgFile) {
+		viper.SetConfigFile(cfgFile)
+	} else {
+		return nil
+	}
+	viper.SetConfigType("toml")
+	viper.AutomaticEnv()
+	if err := viper.ReadInConfig(); err != nil {
+		return err
```

Comment:
ignore `err` when `err`  is `viper.ConfigFileNotFoundError`

Reply:
thank you ,done